action.yml

```diff
@@ -75,7 +83,11 @@ runs:
       shell: bash
       working-directory: repository
       run: |
-        if [ "${{ github.event_name }}" = "pull_request" ]; then \
+        echo "Action trigger: ${{ github.event_name }}";
+
+        if [[ ${{ inputs.branch }} ]]; then
```

Comment:
When testing this, I'm getting a syntax error around here.
![Screen Shot 2023-03-15 at 10 14 16 AM](https://user-images.githubusercontent.com/13966707/225335855-ebcbecf9-bdee-428a-a2d6-1030e48a16e3.png)
https://github.com/searchspring-implementations/twenty-one.test/actions/runs/4427061711/jobs/7764142634

I think this line should be
```
if [ "${{ inputs.branch }}" != "" ]; then
```

Reply:
Your suggestion fixes the error.